### DISCUSSION TITLE
Error in MNISTSuperpixcels

### DISCUSSION THREAD

**Discussion by FatemehAnami** [pyg-team/pytorch_geometric#8744] [Q&A]
Hi there, 
I faced error when running following code in colab :

dataset = MNISTSuperpixels(root='data', train=True, transform=ToSLIC(75, 0.5, sigma=1.5), pre_transform=KNNGraph(k=6))

My torch version is: 
torch version : '2.1.0+cu121'
 torch cluster version : , '1.6.3+pt21cu121'

and the error is 
ImportError: 'knn_graph' requires 'torch-cluster' 

 while I have torch cluster installed on my colab

**Comment by akihironitta** [comment]
I'd double-check whether `import torch_cluster` succeeds. Reference: https://github.com/rusty1s/pytorch_cluster
